tests/components/flick_electric/test_config_flow.py

```diff
@@ -41,48 +47,24 @@ async def test_form(hass: HomeAssistant) -> None:
     assert result["type"] is FlowResultType.FORM
     assert result["errors"] == {}
 
-    with (
-        patch(
-            "homeassistant.components.flick_electric.config_flow.SimpleFlickAuth.async_get_access_token",
-            return_value="123456789abcdef",
-        ),
-        patch(
-            "homeassistant.components.flick_electric.config_flow.FlickAPI.getCustomerAccounts",
-            return_value=[
-                {
-                    "id": "1234",
-                    "status": "active",
-                    "address": "123 Fake St",
-                    "main_consumer": {"supply_node_ref": "123"},
-                }
-            ],
-        ),
-        patch(
-            "homeassistant.components.flick_electric.config_flow.FlickAPI.getPricing",
-            return_value=_mock_flick_price(),
-        ),
-        patch(
-            "homeassistant.components.flick_electric.async_setup_entry",
-            return_value=True,
-        ) as mock_setup_entry,
-    ):
-        result2 = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            {
-                CONF_USERNAME: "test-username",
-                CONF_PASSWORD: "test-password",
-            },
-        )
-        await hass.async_block_till_done()
+    result2 = await hass.config_entries.flow.async_configure(
```

Comment:
I would overwrite result every time as you don't use it afterwards. I've seen mistakes caused by this